test/create_parameters.jl

```diff
@@ -85,14 +85,57 @@ function create_kid_parameters(toml_dict)
     return kid_params
 end
 
+function determine_cloudy_disttypes(NM::Int, default_gamma=true)
```

Comment:
When default is gamma I expect the first distribution to be gamma. Right now there is no way to make the first dist gamma and the rest exp. While sometimes cloud is assumed gamma and rain exp.

Reply:
This was meant to state whether we should use more exponential modes, versus fewer gamma modes. You're right, I assign the largest mode to be the gamma mode by default (this is typical for cloud/rain), but I'll add an additional kwarg so this is an option.